docs/source/markdown/podman-commit.1.md

```diff
@@ -86,6 +87,11 @@ $ podman commit -q --pause=false containerID image-committed
 e3ce4d93051ceea088d1c242624d659be32cf1667ef62f1d16d6b60193e2c7a8
 ```
 
+```
+$ podman commit containerID
+e3ce4d93051ceea088d1c242624d659be32cf1667ef62f1d16d6b60193e2c7a8
+```
+
```

Comment:
I could have sworn this had been done already, but
LGTM

Reply:
Just above we had the example with containerID but I feel it could be nice to have an example without an image name given. WDYT?

Reply:
This is fine.  My previous comment wasn't on this particular line, but in the change in whole.  I would have sworn we had done this earlier, but I must be losing my marbles.